source/_integrations/lm75.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: LM75
+description: Instructions on how to integrate LM75 sensor connected via I2C within Home Assistant.
+ha_category:
+  - DIY
+ha_release: 0.108
+ha_iot_class: Local Polling
+ha_domain: lm75
+---
+
+The `lm75` sensor platform allows you to get the current temperature from a LM75 device connected via I2C.
+
+## Configuration
+
+To use your LM75 sensor in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: lm75
+    name: Kitchen
+    i2c_address: 0x48
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

This applies to all other optional configuration parameters as well.